querybook/webapp/components/StatementExecutionBar/ResultExportDropdown.tsx

```diff
@@ -194,8 +199,17 @@ export const ResultExportDropdown: React.FunctionComponent<IProps> = ({
                     </span>
                 ),
                 onClick: onExportTSVClick,
-                icon: 'copy',
+                icon: 'Copy',
             },
+            ...(hasMetastoresForUpload
+                ? [
+                      {
+                          name: 'Upload Result to Table',
```

Comment:
Create Table from Result? (still not sure but idk if Upload is correct terminology)